packages/grid/x-data-grid/src/tests/rows.DataGrid.test.tsx

```diff
@@ -311,6 +312,7 @@ describe('<DataGrid /> - Rows', () => {
       await waitFor(() => {
         expect(getRow(0)).not.to.have.class('Mui-selected');
       });
+      await microtasks();
```

Comment:
It's not related to this PR, it's just a fix for the flaky test that fails from time to time: https://app.circleci.com/pipelines/github/mui/mui-x/49318/workflows/26e5b314-bb45-4771-8095-330e1f96670b/jobs/282851
It failed again in this PR, this is why I added a fix here.